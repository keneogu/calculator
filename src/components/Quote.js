import React from 'react';

const Quote = () => (
  <div className="quote">
  <p><blockquote>#
	  The view you adopt for yourself profoundly affects the way you lead your life.
	  It can determine whether you become the person you want to be 
	  and whether you accomplish the things you value
  </blockquote></p>
  </div>
)

export default Quote;