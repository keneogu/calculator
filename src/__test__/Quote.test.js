import renderer from 'react-test-renderer';
import React from 'react';
import Quote from '../components/Quote';

it('matches the Quote snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
