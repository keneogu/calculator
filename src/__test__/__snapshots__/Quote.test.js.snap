// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the Quote snapshot 1`] = `
<div
  className="quote"
>
  <p>
    <blockquote>
      The view you adopt for yourself profoundly affects the way you lead your life. It can determine whether you become the person you want to be and whether you accomplish the things you value.
    </blockquote>
  </p>
</div>
`;
